lsp-ui-doc.el

```diff
@@ -61,15 +61,16 @@
   :type 'boolean
   :group 'lsp-ui-doc)
 
-(defcustom lsp-ui-doc-position 'top
+(defcustom lsp-ui-doc-position 'at-point
```

Comment:
Is it intentional change?

Reply:
Yes, @yyoncho suggested in Gitter that we could make it the default. Let's see the feedback we get as people upgrade the package.